tests/data/doc_write/rdf-simple.json

```diff
@@ -61,8 +61,11 @@
           }, 
           "ns1:fileName": "./some/path/tofile"
         }
-      }, 
+      },
+      "ns1:relationship": {"ns1:Relationship": {"@rdf:about": "SPDXRef-DOCUMENT",
+                                        "ns1:relatedSpdxElement": "SPDXRef-Package",
+                                        "ns1:relationshipType": "DESCRIBES"}},
```

Comment:
A comment about these additions:
This structure looks reasonable to me and is apparently what our rdf writer produces at the moment. It may not be what the spec intends, though:

The structure in e.g. [this example](https://github.com/spdx/spdx-spec/blob/v2.3/examples/SPDXRdfExample-v2.2.spdx.rdf.xml#L1412) is different: The related element is usually expanded (making the whole thing completely unreadable), and there doesn't seem to be a reference to the current element (maybe relationships are put on the originating object instead of the top-level document?).

Considering that there are other oddities in this test file as well (e.g. `referencesFile` and `describesPackage` doesn't exist in example files from the spec or java tools), and the documentation on the desired state is unclear, I'd leave it at this for now. I will ask on the ML what's going on with rdf...

Reply:
As far as I know, elements in rdf files are normally expanded at the first place they appear (which does get quite muddy, yeah).